NuGet/Microsoft.ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
```

Comment:
Just making sure this didn't get lost in the file rename:

Probably should include the licenseUrl as well: https://github.com/Microsoft/ChakraCore-Debugger/blob/master/LICENSE

Reply:
Done.

Reply:
I commented just before the rename; we probably also should include a copyright notice.